lmdeploy/cli/lite.py

```diff
@@ -35,7 +35,7 @@ def add_parser_auto_awq():
         ArgumentHelper.calib_seqlen(parser)
         ArgumentHelper.calib_batchsize(parser)
         ArgumentHelper.calib_search_scale(parser)
-        ArgumentHelper.device(parser)
+        ArgumentHelper.device(parser, choices=['cuda', 'cpu'])
```

Comment:
we don't support cpu.

Reply:
it's for `lmdeploy lite` and it could use CPU

Reply:
`lmdeploy lite` doesn't need `--device` now